examples/visualization/plot_roi_erpimage_by_rt.py

```diff
@@ -0,0 +1,94 @@
+"""
+========================================================================
+Plot single trial activity, EEG channels grouped by ROI and sorted by RT
+========================================================================
+
+This will produce what is sometimes called an event related
+potential / field (ERP/ERF) image.
+
+The EEGLAB example file is read in and response times are calculated.
+ROIs are determined by the channel types (in 10/20 channel notation,
+even channels are right, odd are left, and 'z' are central). The
+median within each channel is calculated, and the trials are plotted,
+sorted by response time.
+"""
+# Authors: Jona Sassenhagen <jona.sassenhagen@gmail.com>
+#
+# License: BSD (3-clause)
+
+from numpy import median
```

Comment:
from numpy as np
np.median

the reason is that then np.median is explicit of where it's coming from